readme.txt

```diff
@@ -1,136 +1,141 @@
-=== WP Nasa/ADS Query Importer ===
-Contributors: giodimilia
+=== WP Nasa/ADS Query Importer
```

Comment:
Can you please consider the comments to the readme.md, update the readme.txt accordingly and remove the readme.md? I know the readme.md is nicer for github, but wordpress forces us to use readme.txt:

https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/

And it is better not to have things duplicated, otherwise we need to remember to update twice the same paragraphs.